README.md

```diff
@@ -1,50 +1,50 @@
 # VKD3D-Proton
 
-VKD3D-Proton is a fork of VKD3D, which aims to implement the full Direct3D 12 API on top of Vulkan.
-The project serves as the development effort for Direct3D 12 support in [Proton](https://github.com/ValveSoftware/Proton).
+The VKD3D-Proton is a VKD3D fork designed to implement the entire Direct3D 12 API at the top of Vulcan.
```

Comment:
at the top of Vulcan? Vulcan is a planet in Star Trek. Also, it's indeed *on top* of Vulkan.